dspace/config/spring/api/core-services.xml

```diff
@@ -143,69 +144,147 @@
     </bean>
 
     <util:map id="isAuthorOfPublicationMap">
-        <entry key="dc_contributor_author" value-ref="publicationAuthor_author"/>
+        <entry key="dc.contributor.author" value-ref="publicationAuthor_author"/>
     </util:map>
-    <util:list id="publicationAuthor_author">
-        <value>orgunit.identifier.name</value>
-        <value>person.identifier.lastname</value>
-        <value>person.identifier.firstname</value>
-    </util:list>
+    <bean class="org.dspace.content.virtual.Concatenate" id="publicationAuthor_author">
+        <property name="fields">
+            <util:list>
+                <value>person.identifier.lastname</value>
+                <value>person.identifier.firstname</value>
+            </util:list>
+        </property>
+        <property name="separator">
+            <value>, </value>
+        </property>
+    </bean>
 
     <util:map id="isOrgUnitOfPublicationMap">
         <entry key="dc_contributor_other" value-ref="publicationOrgUnit_name"/>
     </util:map>
-    <util:list id="publicationOrgUnit_name">
-        <value>orgunit.identifier.name</value>
-    </util:list>
+    <bean class="org.dspace.content.virtual.Concatenate" id="publicationOrgUnit_name">
```

Comment:
Because the structure of the `<bean>` definitions in Spring XML configs is getting more and more complex, I'd like to see some inline (XML) comments that describe each of these configurations.  For instead, what is `Concatenate` doing for this bean -- is it ensuring multiple `orgunit.identifier.name` values are comma separated?  In contrast, what is `Concatenate doing for `publicationAuthor_author`? In that situation, it looks to be a concatenation of *separate* fields instead of multiple values.

We really just need to start thinking about commenting these XML configs to make them easier to understand & tweak later as needed.

Reply:
Yes, please, especially when DSpace must provide its own helpers instead of using well-known standard ones.  I would underscore Tim's point as well:  in bean wiring we need to understand, not how the helper works in general, but what its use accomplishes *here*.